python_on_whales/components/compose/cli_wrapper.py

```diff
@@ -651,6 +653,7 @@ def up(
         scales: Dict[str, int] = {},
         attach_dependencies: bool = False,
         force_recreate: bool = False,
+        no_recreate: bool = False,
```

Comment:
Thinking about it a bit longer, the fact that the parameter name is negative (with "no" inside) is a constraint on the command line. In python-on-whales, in other parts of the codebase, we try to make the parameters name positive to be more user friendly.
I know this seems weird because there is "no_build" just below, but i'll deprecate it in the future in favor of `build: bool =True`. 
```suggestion
        recreate: bool = True,
```

Reply:
Sure no problem. I force push the changes, let me know what you think. Many thanks

Reply:
Feel free to organise the commits however you want, I squash them anyway when I merge in master